### DISCUSSION TITLE
KusionStack + KCL 20th Weekly Meeting | 第 20 次周会 (2023.08.17)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#5] [General]
## 会议内容
KusionStack + KCL 开源之夏、GLCC 工作内容同步
KusionStack + KCL 近期开发日常工作同步

## 会议方式
Peefy 邀请您参加腾讯会议
会议主题：KusionStack + KCL 社区双周会
会议时间：2023/08/03 20:00-20:45 (GMT+08:00) 中国标准时间 - 北京
重复周期：2023/08/03-2024/08/29 20:00-20:45, 每两周 (周四)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/Hc6sNpqTWnPb

#腾讯会议：778-2381-6338

复制该信息，打开手机腾讯会议即可参与

**Comment by ArianaKallen** [comment]
Der Fokus liegt offenbar auf dem Austausch über die Fortschritte im Rahmen von Open Source Summer und GLCC sowie auf regelmäßigen Entwicklungsupdates für KusionStack und KCL. Der regelmäßige zweiwöchentliche Rhythmus deutet auf eine aktive und stabile Community hin. Bei diesen Treffen geht es in der Regel weniger um große Ankündigungen, sondern vielmehr um Abstimmung, Hindernisse und Transparenz. Für alle Beteiligten ist es dundeeslots https://casino-sofort.com/online-casinos/dundee-slots/ eine gute Gelegenheit, auf dem Laufenden zu bleiben und die Arbeit zu koordinieren. Selbst wenn man sich nicht aktiv einbringt, hilft allein das Zuhören, die Richtung der Projekte und die Zusammenhänge der verschiedenen Bemühungen zu verstehen.
